cypress/support/step-definitions/grid-steps.js

```diff
@@ -0,0 +1,31 @@
+import { pod, gridPod } from '../../locators/grid';
+
+Then('pod {int} is {string}', (index, title) => {
+  pod(index).should('have.text', title);
+});
+
+When('I resize grid viewport to default', () => {
+  cy.viewport(1958, 900);
+});
+
+When('I resize grid viewport to large', () => {
+  cy.viewport(1500, 900);
+});
+
+When('I resize grid viewport to medium', () => {
+  cy.viewport(1300, 900);
+});
+
+When('I resize grid viewport to small', () => {
+  cy.viewport(900, 900);
+});
+
+Then('pod {int} has height from row {string} to row {string}', (index, rowStart, rowEnd) => {
+  gridPod(index).should('have.css', 'grid-row-start', `${rowStart}`);
```

Comment:
```suggestion
  gridPod(index).should('have.css', 'grid-row-start', `${rowStart}`)
```
`;` needs to be removed to make possible to chain next assertion